salt/modules/cassandra_cql.py

```diff
@@ -0,0 +1,585 @@
+        msg = "ERROR: Cassandra query failed: %s" % query
+        raise CommandExecutionError(msg, e)
+
+    if results:
+        for index in range(0, len(results)):
+            result = results[index]
+            values = {}
+            for key, value in result.iteritems():
+                # Salt won't return dictionaries with odd types like uuid.UUID
+                if not isinstance(value, unicode):
+                    value = str(value)
+                values[key] = value
+            ret.append(values)
+    cluster.shutdown()
+    return ret
+
+
+def version(contact_points=None, port=None, cql_user=None, cql_pass=None):
+    '''
+    Show the Cassandra version.
+
+    :param contact_points: The Cassandra cluster addresses, can either be a string or a list of IPs.
+    :type  contact_points: str | list[str]
+    :param cql_user:       The Cassandra user if authentication is turned on.
+    :type  cql_user:       str
+    :param cql_pass:       The Cassandra user password if authentication is turned on.
+    :type  cql_pass:       str
+    :param port:           The Cassandra cluster port, defaults to None.
+    :type  port:           int
+    :return:               The version for this Cassandra cluster.
+    :rtype:                str
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt 'minion1' cassandra.version
+
+        salt 'minion1' cassandra.version contact_points=minion1
+    '''
+    query = 'select release_version from system.local limit 1'
+    ret = _query(query, contact_points, port, cql_user, cql_pass)
+    if ret:
+        return ret[0].get('release_version')
+    else:
+        return False
+
+
+def info(contact_points=None, port=None, cql_user=None, cql_pass=None):
+    '''
+    Show the Cassandra information for this cluster.
+
+    :param contact_points: The Cassandra cluster addresses, can either be a string or a list of IPs.
+    :type  contact_points: str | list[str]
+    :param cql_user:       The Cassandra user if authentication is turned on.
+    :type  cql_user:       str
+    :param cql_pass:       The Cassandra user password if authentication is turned on.
+    :type  cql_pass:       str
+    :param port:           The Cassandra cluster port, defaults to None.
+    :type  port:           int
+    :return:               The information for this Cassandra cluster.
+    :rtype:                dict
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt 'minion1' cassandra.info
+
+        salt 'minion1' cassandra.info contact_points=minion1
+    '''
+
+    query = 'select cluster_name, \
+             data_center, \
+             partitioner, \
+             host_id, \
+             rack, \
+             release_version, \
+             cql_version, \
+             schema_version, \
+             thrift_version from system.local limit 1'
+    ret = _query(query, contact_points, port, cql_user, cql_pass)
+    if ret:
+        return ret[0]
```

Comment:
That would be more compact:
```python
return ret and ret[0]
```
But please consider to stay consistent across the functions what you return: either `None` or `False` in case of nothing.